src/components/SpreadTrendChart/SpreadTrend.js

```diff
@@ -20,7 +21,11 @@ const drawTrendChart = (sheetTrend, trendChart) => {
     Tested: ["Tested"],
   };
 
-  for (let i = 0; i < sheetTrend.length; i++) {
+  for (
+    let i = sheetTrend.length - CHART_TIME_PERIOD;
```

Comment:
Minor: This is a little bit weird format for a for-loop, but I guess the sheetTrend.length - CHART_TIME_PERIOD is too long?

Reply:
I wrote it as one line, but it looks like the auto-cleanup tool made this change :)